src/CONST.ts

```diff
@@ -2588,7 +2588,7 @@ const CONST = {
         PINK: 'Pink',
     },
 
-    MAP_PADDING: 50,
+    MAP_PADDING: 32,
```

Comment:
```suggestion

```

Let's remove the duplicated `MAP_PADDING` variable and only keep `MAPBOX.PADDING` which we should use in both places where `<DistanceMapView />` is used.